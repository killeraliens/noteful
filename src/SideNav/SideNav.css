.SideNav {
  padding: 16px;
  background-color: lightgrey;
}
