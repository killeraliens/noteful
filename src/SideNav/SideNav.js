import React from 'react'

export default function SideNav(props) {
  //console.log(props.children);
  return(
    <div className='SideNav'>
       {props.children}
    </div>
  )
}
