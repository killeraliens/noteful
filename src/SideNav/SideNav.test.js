import React from 'react';
import ReactDOM from 'react-dom';
//import { Shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import SideNav from './SideNav';

describe('SideNav', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<SideNav/>, div);
    ReactDOM.unmountComponentAtNode(div);
  })

  it('renders its children', () => {
    const div = document.createElement('div');
    const childDiv = document.createElement('div');
    const instance = renderer.create(<SideNav><div>test</div></SideNav>).toJSON();
    console.log(instance);
    expect(instance).toMatchSnapshot()
  })
})
