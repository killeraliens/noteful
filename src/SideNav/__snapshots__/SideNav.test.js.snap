// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideNav renders its children 1`] = `
<div
  className="SideNav"
>
  <div>
    test
  </div>
</div>
`;
